<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>뉴스 원문</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_index">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>-10</y>
      <width>600</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>470</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>다시 듣기</string>
    </property>
    <property name="shortcut">
     <string>0</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>740</width>
      <height>300</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_prev">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>470</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>돌아가기</string>
    </property>
    <property name="shortcut">
     <string>Backspace</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="titleBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>740</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
